### ISSUE TITLE
Use new PhetUnits subsystem

### ISSUE THREAD

**Issue by samreid** [phetsims/quantum-wave-interference#204]
From https://github.com/phetsims/quantum-wave-interference/issues/135, I identified several places that are not using the PhET Units subsystem correctly.

**Comment by samreid** [comment]
@matthew-blackman can you please spot check some units in the a11y view description (like slit separations), and see if they still look correct after the change? If so, you can close

**Comment by matthew-blackman** [comment]
I checked the aria-valuetext for all controls of numbers that have units across all scenes, and everything looks correct. Nice work! Closing.
